Dockerfile

```diff
@@ -32,26 +33,28 @@ RUN git clone https://github.com/nginx-proxy/forego/ \
    && cd - \
    && rm -rf /go/forego
 
+FROM python:${PYTHON_VER} as pythonbuilder
+RUN apt-get update && apt-get install -y build-essential libdbus-1-dev
```

Comment:
```suggestion
RUN apt-get update && apt-get install -y build-essential libdbus-1-dev cmake
```

Reply:
Sure, but why do we need cmake?

Reply:
To build ninja... to build the native components of mdns-publisher on ARMv7 (where there are no pre-built binaries)